docs/user_guide/bank/community_product_drives.md

```diff
@@ -1 +1,25 @@
-Not yet written
\ No newline at end of file
+# Product Drive Participants
+
+You may have organized a product drive in your community and would like to manage the participants.
+
+To manage product drive participants, click on " Product Drive Participants" under the "Community" section.
+
+![Product Drive Participants](images/community/product_drive/product_drive_page.jpg)
+
+If you have done product drives before, the page will be populated with the participants' details. 
+
+To add a new participant, click on the "+ New Product Drive Participant" button, add their details including the business name, contact name, phone, email, address and comments.
+
+![Add Participant](images/community/product_drive/add_paticipant.jpg)
+
+After saving the participant's details there will be a new row in the Product Drive Participants page.
+
+Click the "Edit" button to edit the participant's details. 
```

Comment:
Please provide a subheading "Editing participant information"

Could you provide a screenshot of that screen as well?